docs/source/using/addons.adoc

```diff
@@ -112,6 +112,23 @@ The following table shows the supported variables that are available for interpo
 |Name of the current add-on.
 |===
 
+The commands xref:../command-ref/minishift_start.adoc#[`minishift start`] as well as xref:../command-ref/minishift_addons_apply.adoc#[`minishift addons apply`] also provide a `--addon-env` flag which allow to pass variables for interpolation dynamically, for example:
+
+----
+$ minishift addons apply --addon-env ACME_TOKEN=1234 acme
+----
+
+The `--addon-env` flag can be specified multiple times to define more than one variables for interpolation.
```

Comment:
>What do you mean? This would then again go into variable interpolation of the shell which is a topic outside of the scope of Minishift.

Sure, we do not need to go the details which is out of scope of Minishift, but what about mentioning a single or couple of sentence about it. The idea should be to tell users that this capability also exists. If you are saying that its not a recommended method then it is fine not to mention in the docs.